src/Employee.java

```diff
@@ -0,0 +1,56 @@
+public class Employee {
+    private final Integer id;
+    private String FullName;
```

Comment:
```suggestion
    private String fullName;
```
названия полей надо писать в camelCase, а не CamelCase